### ISSUE TITLE
Running `npm install` in a clean clone of this repo modifies package-lock.json

### ISSUE THREAD

**Issue by yokuze** [silvermine/eslint-config-silvermine#51]
Running `npm install` from a clean state (e.g. right after cloning this repo) modifies the `package-lock.json` file, like so:

```
diff --git a/package-lock.json b/package-lock.json
index 4304a76..02f7223 100644
--- a/package-lock.json
+++ b/package-lock.json
@@ -3784,11 +3784,11 @@
       "dev": true,
       "requires": {
         "chalk": "^2.4.1",
-        "conventional-changelog": "^3.0.6",
-        "conventional-recommended-bump": "^4.0.4",
+        "conventional-changelog": "^3.1.2",
+        "conventional-recommended-bump": "^4.1.1",
         "detect-indent": "^5.0.0",
         "detect-newline": "^2.1.0",
-        "dotgitignore": "^1.0.3",
+        "dotgitignore": "^2.1.0",
         "figures": "^2.0.0",
         "fs-access": "^1.0.0",
         "git-semver-tags": "^2.0.2",
```

**Comment by yokuze** [comment]
This issue is caused by the dependency on a fork of `standard-version`: https://github.com/silvermine/eslint-config-silvermine/blob/887c55e44e68aff5b5b3f8add20b0e99f99783ad/package.json#L38

This fork does not contain a `package-lock.json` file, so when we run `npm install`, dependencies of this dependency are updated the latest version according to the fork's `package.json` file, thus changing our `package-lock.json` file.

This fork of `standard-version` was created to fix [an issue that we had with `standard-version` that caused duplicated headers in the CHANGELOG.md output](https://github.com/conventional-changelog/standard-version/pull/307). Our PR to fix that issue has since been merged, tagged, and released. The fix is available in v6.0.0 of` standard-version`.

Updating to the latest version of `standard-version` should fix this issue.

**Comment by yokuze** [comment]
This was fixed with #52
